src/core/components/hv-route/index.tsx

```diff
@@ -605,4 +643,14 @@ export default function HvRoute(props: Types.Props) {
   );
 }
 
+function HvRouteBehaviorWrapper(props: Types.Props) {
```

Comment:
do we need this additional component, or could the `<BackBehaviorProvider/>` be added to `HvRoute`?

Reply:
It needs to be separate to allow the FC to access the context. I have updated this to rename it as "HvRoute" so that all instances are wrapped correctly.